code/src/index.css

```diff
@@ -1,13 +1,130 @@
-body {
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400;600&display=swap');
+
+* {
+  font-family: 'Roboto Mono', monospace;
+  font-weight: 400;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  padding: 0;
+  -webkit-box-smoothing: border-box;
+  -moz-box-smoothing: border-box;
+  box-sizing: border-box;
   -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeLegibility;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Roboto Mono'
+}
+
+h1 {
+  text-align: center;
+  margin-top: 30px;
+}
+
+
+h2 {
+  text-align: center;
+}
+
+:root {
+    --clr-black: #111;
+    --clr-white: #fff;
+    --clr-light-grey: #e3e1e1;
+    --clr-rose: #ffadad;
+    --clr-rose2: #fd9d9d;
+}
+
+/* ------- NEW THOUGHT --------- */
+
+.thought-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 85vw;
+  margin: 30px;
+  padding:20px 25px 5px;
+  border: 1px solir var(--clr-black);
+  box-shadow: 10px 10px var(--clr-black);
+  background-color: var(--clr-light-grey);
+}
+
+input {
+  display: none;
+}
+
+#thought-input {
+  width: calc(90vw - 90px);
+  font-size: 16px;
+  height: 65px;
+  padding: 10px;
+  margin: 10px auto 5px auto;
+  outline-color: var(--clr--rose);
+}
+
+.newthought-question {
+  font-size: 16px;
+}
+
+/* ------- BUTTON --------- */
```

Comment:
like your comments in CSS so you can easy follow what is styled and fined it the other files :)